lib/utils.pm

```diff
@@ -1111,7 +1111,13 @@ sub handle_login {
     }
     if (check_var('DESKTOP', 'gnome') || (check_var('DESKTOP', 'lxde') && check_var('VERSION', '42.1'))) {
         # DMs in condition above have to select user
-        send_key 'ret';
+        if ((is_sle && sle_version_at_least('15')) || (is_leap && leap_version_at_least('15.0')) || is_tumbleweed) {
```

Comment:
I don't understand how this is supposed to work when being within the `if check_var('VERSION', '42.1')` which should be only true for openSUSE Leap 42.1

Reply:
Statement is true if desktop is gnome, 42.1 version check is only relevant for lxde

Reply:
ah, yes. I see. That should be removed however as we do not have 42.1 tests running anymore. I will do that